website/next.config.mjs

```diff
@@ -2,7 +2,29 @@ import { withGuildDocs } from '@theguild/components/next.config';
 
 export default withGuildDocs({
   redirects: () =>
-    Object.entries({}).map(([from, to]) => ({
+    Object.entries({
+      '/docs/basics/mutations.html': '/docs/data/mutations',
+      '/docs/1.0/basics/setup': '/v1',
+      '/docs/basics/setup.html': '/docs/get-started',
+      '/docs/1.0/features/cache-updates': 'v1/basics/setup',
+      '/docs/1.0/basics/caching': '/v1/basics/queries',
+      '/docs/features/cache-updates': '/docs/caching/configuration',
+      '/docs/Roboto-Regular.ttf': '/docs',
+      '/docs/basics/queries.html': '/docs/data/queries',
+      '/docs/1.0/basics/network-layer': '/v1/basics/network-layer',
```

Comment:
Your should use a dynamic pattern here

Reply:
Found a way to do it :) what do you think?